datafusion/core/tests/sqllogictests/test_files/groupby.slt

```diff
@@ -2760,8 +2772,8 @@ Projection: FIRST_VALUE(sales_global.amount) ORDER BY [sales_global.ts ASC NULLS
 physical_plan
 ProjectionExec: expr=[FIRST_VALUE(sales_global.amount) ORDER BY [sales_global.ts ASC NULLS LAST]@0 as fv1, LAST_VALUE(sales_global.amount) ORDER BY [sales_global.ts ASC NULLS LAST]@1 as fv2]
 --AggregateExec: mode=Final, gby=[], aggr=[FIRST_VALUE(sales_global.amount), LAST_VALUE(sales_global.amount)]
-----SortPreservingMergeExec: [ts@0 ASC NULLS LAST]
```

Comment:
Also `SortPreservingMergeExec` is replaced with `CoalescePartitionsExec`. since in final mode we no longer require any ordering for its input.